### PR TITLE
Adds setup.py files and conda recipes for cugraph-service

### PR DESCRIPTION
**Pull request by BradReesWork**
Closes https://github.com/rapidsai/GaaS/issues/20
Closes https://github.com/rapidsai/GaaS/issues/19

For cugraph-service

- Adding python setup 
- Adding build options to the build.sh script
- creating the build to conda packages

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun tests

File: conda/recipes/cugraph-service/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      run:
+        - python x.x
+        - thriftpy2
+        - {{ pin_compatible('cugraph-service-client', exact=True) }}
+        - {{ pin_compatible('cugraph', exact=True) }}
+        - cupy >=9.5.0,<12.0.0a0
+        - numpy
+        - ucx-py
+        - distributed >=2022.9.2
+        - dask-cuda
```
[the remark this answers is not in the document]
Comment (rlratzel) [reply conda/recipes/cugraph-service/meta.yaml]: No reason, and I agree. I added that (and other pins that should probably be used) in the latest commit.

**Review by raydouglass — APPROVED** [review]

**Review by rlratzel — APPROVED** [review]
I'm approving my own work here because this needs a python-codeowner and Brad can't approve since it started as his PR.  I'll merge after this gets another approval.

**Review by jnke2016 — APPROVED** [review]

**Comment by rlratzel** [comment]
@gpucibot merge

**Pull request merged**